package.json

```diff
@@ -66,6 +66,7 @@
     "npm-run-all": "^4.1.5",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
+    "redux-mock-store": "^1.5.3",
```

Comment:
Who did you get this change approved by? There is no mention of this addition in your PR comments. 
Thanks! :)

Reply:
We got approval from Bryce to install this package in order to write a functioning test! :)

Reply:
Sweet, thank you!